docs/design/2022-03-10-backup-with-batch-create-table.md

```diff
@@ -110,4 +110,4 @@ Integration test also covered among CDC, BR, TiDB, binlog, TiFlash etc.
 - 61259 tables restore takes 4 minute 50 seconds, 200+ tables/per seconds with following configuration:
 TiDB x 1: 16 CPU, 32 GB
 PD x 1: 16 CPU, 32 GB
-TiKV x 3: 16 CPU, 32 GB
\ No newline at end of file
+TiKV x 3: 16 CPU, 32 GB
```

Comment:
What's the difference?

Reply:
IDK... Maybe it is formatted by the tool.